src/main/java/io/vertx/gsoc2023/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -86,7 +91,21 @@ private void setupGet(Router router) {
   }
 
   private HttpServer createHttpServer(Router router) {
-    return vertx.createHttpServer(new HttpServerOptions()).requestHandler(router);
+    return vertx.createHttpServer(new HttpServerOptions()).requestHandler(router)
```

Comment:
You should not create another server, instead, you should create a new route to `/realtime`. Then in the route handler, you can perform a websocket upgrade using methods available on the HTTP server request object.

Reply:
It is the same HttpServer actually! Anyways, it didn't seem to be the "Vert.x Web" idiomatic way to do it but I couldn't come up with a solution using routers before. I'll try again using the tip given

Reply:
Yeah, it works in this case thanks to server load balancing, but it's not a good practice.